server/routes/index.ts

```diff
@@ -15,5 +16,8 @@ export default function routes(services: Services): Router {
     res.redirect('/confirm-arrival/choose-prisoner')
   })
 
+  const temporaryAbsencesController = new TemporaryAbsencesController(services.temporaryAbsencesService)
+  get('/confirm-arrival/return-from-temporary-absence', temporaryAbsencesController.view())
```

Comment:
at some point we should look at maybe extracting out the separate sections of the app into their own routers.. guess there's no push to do that now